src/services/addressResolver/index.ts

```diff
@@ -1,9 +1,7 @@
 import axios, { AxiosResponse } from "axios";
-import { join } from "path";
```

Comment:
Is there an issue with using join from path? 

The custom implementation of join might resolve incorrectly in some cases.
"a/", "/b" => will return "a//b"